Makefile

```diff
@@ -223,3 +223,15 @@ devenv-up: $(INSTALL_TARGET) $(EMILY_LAMBDA_BINARY) $(EMILY_CDK_TEMPLATE)
 devenv-down:
 	cd devenv/local/docker-compose && \
 	CONTAINER_HOST=$(_CONTAINER_HOST) sh down.sh
+
+run-signer:
+	docker compose --file docker-compose.signer.yml down;
+	docker compose --file docker-compose.signer.yml up postgres bitcoind --detach;
```

Comment:
I suppose this is fine for now, but let's add a TODO to clean up the devenv docker compose file so that we can consolidate around that. I think all we need to do is add a profile for the core services that the signer needs, add the signer service, and fix the bitcoin service which is unnecessarily complex.

Reply:
Added in https://github.com/stacks-network/sbtc/pull/533/commits/b84e42c3a97ad954e6f9ddaf25421b07b9a9a9d1 and linked to #536